docs/video/smoothie/recipe.md

```diff
@@ -53,299 +68,342 @@ On the left is a 240FPS, and on the right a 60FPS video with frame blending, thi
 
     Setting the `fps `to 120, 180 or higher depending on how so you wanna go
 
-    And most importantly raising the `blur intensity` to 2.5 or even higher depending on your taste
+    Raising the `blur intensity` to 2.5 or even higher depending on your taste
+
+    And then doing frame blending / smart resampling in the editor
 
 `enabled`: yes
 
-:   Whether or not you wish to enable this setting, if disabled the rest of this category does not matter
+:   Whether or not you wish to enable this setting, if disabled the rest of this category does not matter.
 
 `fps`: 60
 
-:   Output framerate to cap to, this does not affect how the frame blending is applied [in the code](https://github.com/couleur-tweak-tips/smoothie-rs/blob/5bedf4ff231fd56832deacf4e32c5eb9f640c004/target/jamba.vpy#L350)
+:   Output framerate.
+
+<!--
+...this does not affect how the frame blending is applied in the code
+
+wtf does this even mean? people would assume this already. useless info imo
```

Comment:
thinking about this the claim i made is false, because the output fps actually makes smoothie calculate the amount of weights, which is multiplied by the intensity